### PR TITLE
:bowtie:

### PR DESCRIPTION
**Pull request by jsstrn**
Please accept my code :kissing_closed_eyes:

How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible
- 10
Was there anything that you struggled with?
- not sure
Is there anything that you'd like some further information on?
- nope
Roughly, how long did it take?
- 2 hours

### CODE REVIEW

**Comment by jeremiahalex** [comment]
Why have you used a :bowtie: in your comment?

File: students/jesstern-rays/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+###Your Name
+- You can add a little description here if you want. What you're currently working on, how you're feeling etc. Remember it's public though.
+
+Homework
+
+Home work name
+0. Lab1
```
Comment (jsstrn) [inline students/jesstern-rays/README.md]: :grey_question:

File: students/jesstern-rays/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+0. Lab2
+0. Lab3
+
+```javascript
+var x = 1
+```
+Lab name
```
Comment (jsstrn) [inline students/jesstern-rays/README.md]: :-1:

File: students/jesstern-rays/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+Projects
+
+Project name
```
Comment (jsstrn) [inline students/jesstern-rays/README.md]: @jeremiahalex @lanbau

**Pull request merged**